### PR TITLE
docs: correct auto update PR section

### PR DESCRIPTION
**Pull request by zdtsw**
running every 30mins, not nightly

**Can wait after release done to review**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,18 +52,25 @@ git checkout main
...
 
-This script will create a file called _eclipse-temurin_.
+This script will create a file called _eclipse-temurin_ by default.
 
-Then edit the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin), replacing all of the contents.
+Create new PR to replace [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with new content of _eclipse-temurin_ 
 
-At the bottom of that edit screen add a title and description for the commit and click on the _Propose Change_ button.
+- Go to https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin web UI 
+- Cliek **edit(pencil button)**
```
Comment (smlambert) [inline README.md:60]:
```suggestion
- Click **edit(pencil button)** 
```

File: README.md [hunk 2]
```diff
@@ -52,18 +52,25 @@ git checkout main
...
 
-Then edit the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin), replacing all of the contents.
+Create new PR to replace [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with new content of _eclipse-temurin_ 
 
-At the bottom of that edit screen add a title and description for the commit and click on the _Propose Change_ button.
+- Go to https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin web UI 
+- Cliek **edit(pencil button)** 
+- Remove its content
+- Copy-paste _eclipse-temurin_'s content
+- At the bottom of that edit screen' Propose changes seciont
```
Comment (smlambert) [inline README.md:63]:
```suggestion
- At the bottom of that edit screen' Propose changes section
```

**Review by smlambert — COMMENTED** [review]
A few typos to correct.  BTW, this would have made a good first issue (which we are now out of) for Outreachy applicants.  If you notice other issues similar to this, please consider flagging it for Outreachy participation.

File: README.md [hunk 3]
```diff
@@ -52,18 +52,25 @@ git checkout main
 ./dockerhub_doc_config_update.sh
 ```
 
-This script will create a file called _eclipse-temurin_.
+This script will create a file called _eclipse-temurin_ by default.
 
-Then edit the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin), replacing all of the contents.
+Create new PR to replace [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with new content of _eclipse-temurin_
```
Comment (gdams) [inline README.md:57]:
```suggestion
Create new PR to replace the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with new content of _eclipse-temurin_ 
```

File: README.md [hunk 4]
```diff
@@ -52,18 +52,25 @@ git checkout main
 ./dockerhub_doc_config_update.sh
 ```
 
-This script will create a file called _eclipse-temurin_.
+This script will create a file called _eclipse-temurin_ by default.
 
-Then edit the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin), replacing all of the contents.
+Create new PR to replace the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with new content of _eclipse-temurin_
```
Comment (gdams) [inline README.md:57]:
```suggestion
Create new PR to replace the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin) with the new contents of _eclipse-temurin_ 
```

**Review by karianna — APPROVED** [review]
LGTM

**Review by gdams — APPROVED** [review]

**Pull request merged**